neuro_evolution_ctrnn/tools/configurations.py

```diff
@@ -43,6 +44,23 @@ class IBrainCfg(abc.ABC):
     use_bias: bool
 
 
+@attr.s(slots=True, auto_attribs=True, frozen=True)
+class IPytorchBrainCfg(IBrainCfg):
+    num_layers: int
+    hidden_size: int
```

Comment:
Can this parameter really be used? In LSTM the hidden_size equals the output size right? So that would be dependent on the environment and would cause errors when defining it explicitly

Reply:
Yes, by adding an additional output matrix at the end of the pytorch brain. The parameter is named `hidden_size` in pytorch too. It isn't nativly in the pytorch brain, because this way it is easier to link multiple networks behind each other.